data/scenarios/02Tutorial/02-grab-place.yaml

```diff
@@ -0,0 +1,52 @@
+name: Grab and place
+description: |
+  Learn how to interact with the world by grabbing entities and placing them.
+entities:
+  - name: goal
+    display:
+      attr: device
+      char: 'X'
+    description:
+    - |
+      !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+      TODO: move twice, grab, go back, place
+      !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+  - name: simple flower
+    display:
+      attr: flower
+      char: '*'
+    description:
+    - A flower.
+    properties: [known]
+
+win: |
+  try {
+    // teleport 0,0
+    e <- grab;
```

Comment:
Why not do `ishere "flower"` instead of grabbing it?

Reply:
Because I forgot about `ishere`, thanks for the suggestion :+1: